Report doesn't state failed files, failed spellings

At the end of the report is

```
>> 142 files are free from spelling errors
```

How are about how many files are not free from spelling errors?

How many spelling mistakes are detected?

I've just been caught out by this one ...

```
npm test

> Scott-Logic-Blogs@0.0.0 test /home/ubuntu/blog
> mdspell "*/_posts/**/*.md" --en-gb --ignore-acronyms --ignore-numbers -r -x

    ceberhardt/_posts/2015-12-07-angular-2.md
 

>> 149 files are free from spelling errors
npm ERR! Test failed.  See above for more details.

npm test returned exit code 1
```

--- a/es5/multi-file-processor.js
+++ b/es5/multi-file-processor.js
@@ -37,9 +37,9 @@ exports.default = function (inputPatterns, options, fileCallback, resultCallback
           return _spellcheck2.default.addWord(word, true);
         });
 
-        fileCallback(file, src, function () {
+        fileCallback(file, src, function (err, result) {
           _spellcheck2.default.resetTemporaryCustomDictionary();
-          fileProcessed();
+          fileProcessed(err, result);
         });
       });
     }, resultCallback);

--- a/es6/multi-file-processor.js
+++ b/es6/multi-file-processor.js
@@ -39,9 +39,9 @@ export default function(inputPatterns, options, fileCallback, resultCallback) {
           spellConfig.getFileWords(file)
             .forEach((word) => spellcheck.addWord(word, true));
 
-          fileCallback(file, src, () => {
+          fileCallback(file, src, (err, result) => {
             spellcheck.resetTemporaryCustomDictionary();
-            fileProcessed();
+            fileProcessed(err, result);
           });
         });
       }, resultCallback);

